platforms/core-configuration/groovy-support/src/testFixtures/java/org/gradle/api/internal/groovy/support/TransformCompoundAssignments.java

```diff
@@ -29,5 +29,5 @@
  */
 @Retention(RetentionPolicy.SOURCE)
 @Target({ElementType.METHOD, ElementType.TYPE})
-@GroovyASTTransformationClass("org.gradle.api.internal.provider.TestCompoundAssignmentTransform")
+@GroovyASTTransformationClass("org.gradle.api.internal.groovy.support.CompoundAssignmentTransformInTest")
 public @interface TransformCompoundAssignments {}
```

Comment:
💅 Could we also att Test prefix to this name, so it's easier to know it comes from a test fixture? E.g.:
`TransformCompoundAssignments` -> `TestTransformCompoundAssignments`